src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -258,6 +259,39 @@ public async Task<MutableConflictResolution> ResolveConflictsAsync()
                 }
             }
 
+            private async Task<Solution> AnnotateRenamedSymbolsAsync()
+            {
+                var solutionWithRenameSymbolAnnotations = _renameLocationSet.Solution;
+
+                if (RenameSymbolAnnotation.ShouldAnnotateSymbol(_renameLocationSet.Symbol))
+                {
+                    // Add the RenameSymbolAnnotation to declarations that will change
+                    foreach (var syntaxReferenceGroup in _renameLocationSet.Symbol.DeclaringSyntaxReferences.GroupBy(r => r.SyntaxTree))
+                    {
+                        var syntaxTree = syntaxReferenceGroup.Key;
+                        var syntaxReferences = syntaxReferenceGroup.AsArray();
+
+                        var document = solutionWithRenameSymbolAnnotations.GetRequiredDocument(syntaxTree);
+                        var documentEditor = await DocumentEditor.CreateAsync(document, _cancellationToken).ConfigureAwait(false);
+
+                        foreach (var syntaxReference in syntaxReferences)
+                        {
+                            var node = await syntaxReference.GetSyntaxAsync(_cancellationToken).ConfigureAwait(false);
+
+                            if (RenameSymbolAnnotation.TryAnnotateNode(node, _renameLocationSet.Symbol, out var annotatedNode))
+                            {
+                                documentEditor.ReplaceNode(node, annotatedNode);
+                            }
+                        }
+
+                        var annotatedDocument = documentEditor.GetChangedDocument();
+                        solutionWithRenameSymbolAnnotations = annotatedDocument.Project.Solution;
+                    }
+                }
+
+                return solutionWithRenameSymbolAnnotations;
+            }
```

Comment:
@jasonmalinowski ?

Reply:
@CyrusNajmabadi What's the question?

Reply:
you wrote rename.  i need someone with knowledge of the conflict resolver looking at this since i have no idea how it works :D